Bindings/Python/examples/convert_muscle_type.py

```diff
@@ -0,0 +1,111 @@
+# ----------------------------------------------------------------------- #
+# The OpenSim API is a toolkit for musculoskeletal modeling and           #
+# simulation. See http://opensim.stanford.edu and the NOTICE file         #
+# for more information. OpenSim is developed at Stanford University       #
+# and supported by the US National Institutes of Health (U54 GM072970,    #
+# R24 HD065690) and by DARPA through the Warrior Web program.             #
+#                                                                         #
+# Copyright (c) 2005-2016 Stanford University and the Authors             #
+# Author(s): Ayman Habib                                                  #
+# Contributor(s):                                                         #
+#                                                                         #
+# Licensed under the Apache License, Version 2.0 (the "License");         #
+# you may not use this file except in compliance with the License.        #
+# You may obtain a copy of the License at                                 #
+# http://www.apache.org/licenses/LICENSE-2.0.                             #
+#                                                                         #
+# Unless required by applicable law or agreed to in writing, software     #
+# distributed under the License is distributed on an "AS IS" BASIS,       #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         #
+# implied. See the License for the specific language governing            #
+# permissions and limitations under the License.                          #
+# ----------------------------------------------------------------------- #
+
+# convert_muscle_type.py
+# Author: Ayman Habib
+# ------------------------------------------------------------------------#
+# ABSTRACT: This short piece of OpenSim python API example demonstrates   #
+# how to load a model in the OpenSim python API, cycle through the        #
+# the muscles in the model and replace those of type Thelen2003 with      #
+# the more modern (true to publication) Millard2012AccelerationMuscle     #
+# There are differencs between the two muscle models so the resulting     #
+# model may need to be tuned manually.
```

Comment:
I suggest also printing this message to the console so that users don't miss it. Perhaps also specifically state that all the new Millard2012EquilibriumMuscles will have default curves.